### PR TITLE
ci: update pr_check

### PR DESCRIPTION
**Pull request by akhil-jha**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
https://media.giphy.com/media/102mqDgAb4Kfug/giphy.gif

File: .rhcicd/pr_check.sh [hunk 1]
```diff
@@ -15,22 +15,26 @@ COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-fronten
...
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+# shellcheck source=/dev/null
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
-# teardown_docker
-exit $BUILD_RESULTS
+# Run ui tests
+source "${CICD_ROOT}/deploy_ephemeral_env.sh"
+export COMPONENT_NAME="provisioning-backend"
```
Comment (ezr-ondrej) [inline .rhcicd/pr_check.sh:38]: so we test frontend first and then backend as well? :thinking:

Comment (akhil-jha) [reply .rhcicd/pr_check.sh:38]: So the first `COMPONENT_NAME` is used in order to get the eph env. It's passed as `provisioning-frontend=<sha>` in bonfire deploy

The second `COMPONENT_NAME` is used to run the deploy-iqe-cji, Which is backend. And we pass --selenium to let it know it's a UI.

**Review by mshriver — APPROVED** [review]

File: .rhcicd/pr_check.sh (hunk 1 above)
Comment (ezr-ondrej) [reply .rhcicd/pr_check.sh:38]: Thanks for the info! :tada:

**Review by ezr-ondrej — APPROVED** [review]
I see green, happy to find out how it actually works, but for now I just believe you guys know what you're doing ^_^
Thanks @akhil-jha !! Nice work.

**Pull request merged**